extension/test/e2e/pageObjects/AffirmInitSessionFormPage.js

```diff
@@ -0,0 +1,83 @@
+import InitSessionFormPage from './InitSessionFormPage.js'
+
+export default class AffirmInitSessionFormPage extends InitSessionFormPage {
+  async initPaymentSession({ clientKey, paymentAfterCreateSession }) {
+    await super.initPaymentSession(clientKey, paymentAfterCreateSession)
+    await this.pasteValuesInAffirmWebComponent()
+    await this.confirmAffirmWebComponent()
+  }
+
+  async pasteValuesInAffirmWebComponent() {
+    await this.page.waitForSelector('.adyen-checkout__button--pay') // wait for rendering of web component
+
+    const createSessionInputFormElement = await this.page.$(
```

Comment:
I think we could rename **createSession** here, the naming can be something else like: **adyenCheckoutInputFormElement**?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1064/commits/a77571e0a5e21dfb1fd8da8470085809f70df006